\begin{center}
Gabe Morris
\end{center}

In [1]:
import sympy as sp
import matplotlib.pyplot as plt
from IPython.display import display, Latex

plt.style.use('maroon.mplstyle')

s, t = sp.symbols('s t')

display_latex = lambda text: display(Latex(text))

\pagebreak
\tableofcontents
\pagebreak

\begin{center}
\begin{tabular}{c c c}
ME 6643 & Homework 5 & Gabe Morris \\
& & gnm54
\end{tabular}
\end{center}

# Problem 9.5 Part A
## Given
$T(s)=\frac{Y(s)}{F(s)}=\frac{75}{14s+18}$ \
$f(t)=10\sin(1.5t)$

## Find
The steady state response $y_{ss}(t)$

## Solution
$y_{ss}(t)=10|T(j\omega)|\sin(1.5t+\angle T(j\omega))$

In [2]:
T_s = 75/(14*s + 18)
T_s

75/(14*s + 18)

In [3]:
T_jw = T_s.subs(s, sp.I*1.5)
T_jw

0.0980392156862745*(18 - 21.0*I)

In [4]:
mag = sp.Abs(T_jw)
mag

2.71163072273320

In [5]:
angle = sp.arg(T_jw)
angle

-0.862170054667226

In [6]:
y_t = 10*mag*sp.sin(1.5*t + angle)
y_t

27.116307227332*sin(1.5*t - 0.862170054667226)

## Answer
$y_{ss}(t)=27.1\sin(1.5t-0.862)$

\pagebreak

\begin{center}
\begin{tabular}{c c c}
ME 6643 & Homework 5 & Gabe Morris \\
& & gnm54
\end{tabular}
\end{center}

# Problem 9.5 Part C
## Given
$T(s)=\frac{Y(s)}{F(s)}=\frac{s+50}{s+150}$ \
$f(t)=15\sin(100t)$

## Find
The steady state response $y_{ss}(t)$

## Solution
Do the same procedure as above. No need for tables in the book. Python makes this very easy.

In [7]:
T_s = (s + 50)/(s + 150)
T_s

(s + 50)/(s + 150)

In [8]:
T_jw = T_s.subs(s, sp.I*100)
T_jw

(50 + 100*I)*(150 - 100*I)/32500

In [9]:
mag = sp.Abs(T_jw)
mag.n()

0.620173672946042

In [10]:
angle = sp.arg(T_jw)
angle.n()

0.519146114246523

In [11]:
y_t = 15*mag*sp.sin(100*t + angle.n())
y_t.n()

9.30260509419063*sin(100*t + 0.519146114246523)

## Answer
$y_{ss}(t)=9.3\sin(100t+0.519)$